server/protocols/jmap/src/main/java/org/apache/james/jmap/model/OldKeyword.java

```diff
@@ -28,6 +28,14 @@
 import com.google.common.base.Objects;
 
 public class OldKeyword {
+    public static Optional<OldKeyword> computeOldKeywords(Optional<Boolean> isUnread, Optional<Boolean> isFlagged, Optional<Boolean> isAnswered,
```

Comment:
It's going to be so easy to miss up in the order

Reply:
yes but I expect this code to be deleted in some weeks, so no need to invest more on this (more than good unit test to ensure this error does not happen)

Reply:
I propose you a solution with something looking like a builder.
I might have rush a little bit on naming/testing.
If you like it feel free to improve it